worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/WorldGuardEntityListener.java

```diff
@@ -592,6 +572,11 @@ public void onExplosionPrime(ExplosionPrimeEvent event) {
                 event.setCancelled(true);
                 return;
             }
+        } else if (event instanceof AbstractWindCharge) {
```

Comment:
This appears to be a typo^ `event` where you meant `event.getEntity()`
(Thanks to hyper in the Denizen discord who happened to look at this while investigating something else)